### ISSUE TITLE
Suits Menu: allow changing names

### ISSUE THREAD

**Issue by Tkachov** [Tkachov/Overstrike#29]
Allow the user to rename suits the way they'd like. Entered names would be stored in the same .json, and the "install" process should be updating corresponding .localization entries.

**Comment by Tkachov** [comment]
@Coolmuzt , the feature described here would just change the text shown in game. It won't allow to "clone" suits, as game uses their internal ID for that.

Now, while it's probably possible to make a feature that does change internal ID, allowing "cloning", it's a bit tricky to simply visualize in the UI (as this needs to be done before using Suits Menu). In addition to that, I think it's a very niche use case, as there's not much modular suits. If you want, you may create a separate issue describing this idea, but I'm not sure it'd be implemented any time soon.

**Comment by Coolmuzt** [comment]
> [@Coolmuzt](https://github.com/Coolmuzt) , the feature described here would just change the text shown in game. It won't allow to "clone" suits, as game uses their internal ID for that.
> 
> Now, while it's probably possible to make a feature that does change internal ID, allowing "cloning", it's a bit tricky to simply visualize in the UI (as this needs to be done before using Suits Menu). In addition to that, I think it's a very niche use case, as there's not much modular suits. If you want, you may create a separate issue describing this idea, but I'm not sure it'd be implemented any time soon.

I will, thanks. I hope it can be implemented someday, I believe a lot of people would appreciate this feature since there were tons of comments asking for multiple custom Spideys in Tango's mod page.
